### DISCUSSION TITLE
Creating custom themes: nativescript-theme or @nativescript/theme-switcher

### DISCUSSION THREAD

**Discussion by dlcole** [NativeScript/NativeScript#9697] [Q&A]
I posted [this issue](https://github.com/NativeScript/theme/issues/297) in the Nativescript-theme repo but I don't know how much visibility it will get there.  My question is simple enough - Is it possible to create custom themes that are useable with NativeScript 8.1?  If so, are there corresponding instructions?  

The Nativescript-theme readme advertises this feature but the instructions there don't match the tooling and repeated attempts have not yielded a working theme.

**Comment by NathanWalker** [comment]
Hi @dlcole have you seen this? https://blog.nativescript.org/dynamically-swap-themes-with-theme-switcher

**Reply by dlcole** [reply]
@NathanWalker - no, not yet, but I'll check it out and let you know.  Thanks for the prompt reply

**Reply by dlcole** [reply]
So theme-switcher allows me to change themes during the execution of the app, but that's not what I'm looking for.  I want to create a custom theme and then apply that theme at build time and have it used and the nativescript-theme variables (--light-primary, etc) set.  

This gets back to my original question: is there any current documentation for creating and applying a custom theme using the latest versions of NativeScript (8.1.5, JavaScript project) and Nativescript-theme (3.0.1)?
